pkg/applicationprofilemanager/v1/applicationprofile_manager.go

```diff
@@ -607,6 +609,7 @@ func (am *ApplicationProfileManager) startApplicationProfiling(ctx context.Conte
 		SeccompProfilePath:     am.k8sObjectCache.GetSharedContainerData(container.Runtime.ContainerID).SeccompProfilePath,
 		ContainerType:          am.k8sObjectCache.GetSharedContainerData(container.Runtime.ContainerID).ContainerType,
 		ContainerIndex:         am.k8sObjectCache.GetSharedContainerData(container.Runtime.ContainerID).ContainerIndex,
+		PreRunningContainer:    preRunning,
```

Comment:
Why not setting this also on the ContainerWatcher in the SharedContainerData instead of doing it in each manager?